tests/jax/distributed_configs_helper.py

```diff
@@ -8,14 +8,21 @@
 from transformer_engine.jax.fused_attn import AttnBiasType, AttnMaskType\
 
 
-class ShardingConfigs(object):
+class DistributedConfigsHelper(object):
     
     def __init__(self, num_gpus=len(jax.devices())):
         super().__init__()
-        if num_gpus < 2:
-            raise ValueError(f"ShardingConfig: Need at least 2 GPUs.")
-        
         self.device_count = min(num_gpus, 8)
+        if self.device_count < 2:
+            self.layernorm_refs = None
+            self.softmax_types = None
```

Comment:
```suggestion
            self.softmax_types = None
            self.softmax_refs = None
```

Reply:
I just saw this CI failure a few mins ago and was about to fix it but looks like you beat me to it. Thanks! 😆